### PR TITLE
Documentation on migrating email to Gmail CSE

### PR DESCRIPTION
**Pull request by bgrieder**
Documentation only, no code

This migration does not require the privileged endpoints.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,41 +64,41 @@ docker run -p 9998:9998 --name kms ghcr.io/cosmian/kms:4.18.0
...
 ```
 
 **Example**
 
-1. Create a 256-bit symmetric key 
+1. Create a 256-bit symmetric key
 
 ```sh
-➜ cargo run --bin ckms -- sym keys create --number-of-bits 256 --algorithm aes --tag project1
+➜ cargo run --bin ckms -- sym keys create --number-of-bits 256 --algorithm aes --tag ny-file-key
```
Comment (Manuthor) [inline README.md:80]:
```suggestion
➜ cargo run --bin ckms -- sym keys create --number-of-bits 256 --algorithm aes --tag my-key-file
```
?

File: documentation/docs/google_cse/migrating.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+LOG FILE WRITTEN TO
+  ./log.txt
+```
+
+### Export Data from GMail
+
+If you need to exportd data from Gmail, the easiest is to use TakeOut, which you can find
+[here](https://knowledge.workspace.google.com/kb/how-to-export-a-single-user-data-000009160)
+
+To export data for a single user, followz
```
Comment (Manuthor) [inline documentation/docs/google_cse/migrating.md:109]:
```suggestion
To export data for a single user, follow
```

File: documentation/docs/google_cse/migrating.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+PARTIAL SUCCESS
+
+LOG FILE WRITTEN TO
+  ./log.txt
+```
+
+### Export Data from GMail
+
+If you need to exportd data from Gmail, the easiest is to use TakeOut, which you can find
```
Comment (Manuthor) [inline documentation/docs/google_cse/migrating.md:106]:
```suggestion
If you need to export data from Gmail, the easiest is to use TakeOut, which you can find
```

File: README.md (hunk 1 above)
Comment (Manuthor) [inline README.md:80]:
```suggestion
➜ cargo run --bin ckms -- sym keys create --number-of-bits 256 --algorithm aes --tag my-key-file
```

File: documentation/docs/index.md [hunk 4]
```diff
@@ -21,25 +24,44 @@ and an application-level encrypted database indexed with Findex.
...
+    Pre-built binaries, for both the server and CLI (called `ckms`) are available for multiple 
+    operating systems on [Cosmian packages](https://package.cosmian.com/kms/4.18.0/).
+
+    Using [`ckms`](./cli/cli.md), you can easily manage the server: 
 
+    1) Create a 256-bit symmetric key
+    
     ```sh
-    curl http://localhost:9998/version
+    ckms sym keys create --number-of-bits 256 --algorithm aes --tag my-file-key
```
Comment (ccorsin) [inline documentation/docs/index.md:35]: ` --number-of-bits 256 --algorithm aes` these are default argument values - omit them to make it simple ?

Comment (bgrieder) [reply documentation/docs/index.md:35]: You are right, but I prefer to be explicit.
If not, without further explanation, it looks either magic or hard-coded.

**Pull request merged**